packages/xstate-inspect/src/browser.ts

```diff
@@ -7,7 +7,12 @@ import {
   Observer
 } from 'xstate';
 import { XStateDevInterface } from 'xstate/lib/devTools';
-import { toSCXMLEvent, toEventObject, toObserver } from 'xstate/lib/utils';
+import {
+  toSCXMLEvent,
+  toEventObject,
+  toObserver,
+  interopSymbols
```

Comment:
There might be compatibility issues between mismatched versions of `xstate` and `@xstate/inspect` - I would just touch the core directory as part of this PR and revert the changes here.

Reply:
How would you suggest to do this? `InspectReceiver` of `@xstate/inspect` is an alias for an `ActorRef`, which is impacted by these changes in the core.

Reply:
We can revise these in a follow-up PR.